jetstream/tasks.py

```diff
@@ -137,35 +132,67 @@ def send_reports():
     if failed_reports != 0:
         raise Exception("%s/%s reports failed to send to TAS" % (failed_reports, count))
 
+
 @task(name="update_snapshot")
-def update_snapshot():
+def update_snapshot(start_date=None, end_date=None):
     if not settings.USE_ALLOCATION_SOURCE:
         return False
-    allocation_source_total_compute = {}
-    allocation_source_total_burn_rate = {}
-    end_date = timezone.now()
-    for source in AllocationSource.objects.order_by('source_id'):
-        # iterate over user + allocation_source combo
-        for user_allocation_source in UserAllocationSource.objects.filter(allocation_source__exact=source.id).order_by('user__username'):
-            user = user_allocation_source.user
-            # determine end date and start date using last snapshot
-            start_date = user.date_joined
-            # calculate compute used and burn rate for the user and allocation source combo
-            compute_used, burn_rate = total_usage(user.username,start_date,allocation_source_name=source.name,end_date=end_date,burn_rate=True)
-
-            allocation_source_total_compute[source.name] = allocation_source_total_compute.get(source.name,0) + compute_used
-            allocation_source_total_burn_rate[source.name] = allocation_source_total_burn_rate.get(source.name,0) + burn_rate
-
-            payload_ubr = {"allocation_source_id":source.source_id, "username":user.username, "burn_rate":burn_rate, "compute_used":compute_used}
-            EventTable.create_event("user_allocation_snapshot_changed", payload_ubr, user.username)
```

Comment:
This event was removed and is not re-added via hook or other means. Is the intent here to remove this event from being created (to avoid table bloat)?